types/templates.go

```diff
@@ -372,11 +372,14 @@ type ValidatorPageData struct {
 	WithdrawalCount                          uint64
 	SlashingsCount                           uint64
 	PendingCount                             uint64
-	SyncCount                                uint64
-	ScheduledSyncCount                       uint64
-	ParticipatedSyncCount                    uint64
-	MissedSyncCount                          uint64
-	OrphanedSyncCount                        uint64
+	SyncCount                                uint64 // amount of sync committees the validator was (and is) part of
+	SlotsPerSyncCommittee                    uint64
+	SlotsDoneInCurrentSyncCommittee          uint64
+	SyncCountSlots                           uint64
+	ScheduledSyncCountSlots                  uint64
+	ParticipatedSyncCountSlots               uint64
+	MissedSyncCountSlots                     uint64
+	OrphanedSyncCountSlots                   uint64
 	UnmissedSyncPercentage                   float64     // missed/(participated+orphaned)
```

Comment:
Is this comment supposed to say
`// participated/(participated+missed)`
?

Reply:
Changed